include/zephyr/drivers/usb/udc_buf.h

```diff
@@ -70,7 +70,7 @@
  */
 #define UDC_STATIC_BUF_DEFINE(name, size)					\
 	static uint8_t Z_UDC_BUF_SECTION __aligned(UDC_BUF_ALIGN)		\
-	name[ROUND_UP(size, UDC_BUF_GRANULARITY)];
+	name[ROUND_UP(size, UDC_BUF_GRANULARITY)]
```

Comment:
Did you intend to include this commit here? Asking since it's not mentioned in the PR description.

Reply:
I fixed it for you.